src/main.ts

```diff
@@ -86,33 +85,31 @@ const clientParams = {
 
 async function run(): Promise<void> {
   try {
-    console.log(`Current branch: ${branchNameHead}.`)
-    console.log(commandToRun)
-    safeExec(commandToRun)
+    safeExec(`/usr/bin/git fetch`)
+    safeExec(`/usr/bin/git checkout ${branchNameBase}`)
+    safeExec(`/usr/bin/git checkout ${branchNameHead}`)
+
+    const commandToRunOnHead = `npx jest --ci --runInBand --coverage --changedSince=${branchNameBase} --collectCoverage=true --coverageDirectory='./' --coverageReporters="json-summary"`
+    safeExec(`/usr/bin/git branch --show-current`)
+    console.log(commandToRunOnHead)
+    safeExec(commandToRunOnHead)
 
     const codeCoverageNew = <CoverageReport>(
       JSON.parse(fs.readFileSync('coverage-summary.json').toString())
     )
+    console.log('codeCoverageNew', codeCoverageNew)
```

Comment:
I like having these to see the changes on the PR and base. Really they shouldn't add to much bulk now that we are diffing with master, not running on all tests.

Reply:
Open to removing them though.